extensions/blocks/revue/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export const name = 'revue';
+
+export const title = __( 'Revue', 'jetpack' );
+
+export const icon = 'email-alt';
+
+export const settings = {
+	title,
+	description: __( 'Add a Revue signup form.', 'jetpack' ),
+	icon,
+	category: 'jetpack',
+	support: {
+		html: false,
+	},
+	attributes: {
+		revueUsername: {
+			type: 'string',
+		},
+		text: {
+			type: 'string',
+			default: _x( 'Subscribe', 'verb: e.g. subscribe to a newsletter.', 'jetpack' ),
+		},
+		backgroundColor: {
+			type: 'string',
+		},
+		textColor: {
+			type: 'string',
+		},
+		customBackgroundColor: {
+			type: 'string',
+		},
+		customTextColor: {
+			type: 'string',
+		},
+		borderRadius: {
+			type: 'number',
+		},
+		gradient: {
+			type: 'string',
+		},
+		customGradient: {
+			type: 'string',
+		},
+	},
+	edit,
+	save: () => {},
```

Comment:
Maybe we should add a URL to this, so that there is still something to see when this is loaded outside of the editor. I couldn't find a URL that would work though...

Reply:
That'd be nice, but as far as I could see, we need more info than just the username to build a meaningful link to a newsletter.
E.g.
- `https://www.getrevue.co/profile/USERNAME/add_subscriber` works for the form, but not by opening it directly.
- There's an URL for specific issues of a newsletter (`https://www.getrevue.co/profile/USERNAME/issues/ISSUE_TITLE`), but if you remove the issue title (`/USERNAME/issues`) the URL doesn't work anymore.

Reply:
Would this be acceptable to have a string like "Check my newsletter" linking to `https://www.getrevue.co/profile/USERNAME `?